app_flutter/lib/state/agent.dart

```diff
@@ -81,7 +81,7 @@ class AgentState extends ChangeNotifier {
           .fetchAgentStatuses()
           .then((CocoonResponse<List<Agent>> response) {
         if (response.error != null) {
-          print(response.error);
+          print(response.error); // TODO(ianh): report this rather than printing it
```

Comment:
I believe these TODOs should also link to https://github.com/flutter/flutter/issues/52697 (I don't know if flutter/cocoon has that enabled for the linter)

Reply:
Oh interesting I was imagining they'd just get logged to a snackbar or some such. Logging seems good too though.

Reply:
updated the TODO, will commit on green